docs/src/main/paradox/features/linking.md

```diff
@@ -1,5 +1,4 @@
-Linking
--------
+# Linking
```

Comment:
@eed3si9n Is the reason to use H2 for top-level titles to support PDFs output (#14)?